release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -679,6 +679,11 @@ For more information, see xref:../operators/operator-reference.adoc#about-node-t
 You must still use the `performance-addon-operator-must-gather` image when running the `must-gather` command with the Performance Profile Creator. For more information, see xref:../scalability_and_performance/cnf-create-performance-profiles.adoc#gathering-data-about-your-cluster-using-must-gather_cnf-create-performance-profiles[Gathering data about your cluster using must-gather]
 ====
 
+[id="ocp-4-11-PAO-to-NTO-docs"]
+==== Low latency tuning documentation updates
+
+In earlier version of {product-title}, documentation for low latency tuning included references to the Performance Addon Operator. Since the Node Tuning Operator now provides low latency tuning, the documentation title changed from "Performance Addon Operator for low latency nodes" to "Low latency tuning", and multiple cross-references to this document were updated accordingly. See xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#cnf-low-latency-tuning[Low latency tuning] for details.
```

Comment:
Recast the last sentence as: "For more information, see xref..."

Reply:
Done.